woocommerce/woocommerce-android
# Issue/717 illegal state epilogue

Fixes #717 - uses `commitAllowingStateLoss` to prevent the `IllegalStateException` caused by showing the upgrade dialog while the app is backgrounded.

To test:
* Pull `develop` and change [this line](https://github.com/woocommerce/woocommerce-android/blob/develop/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/LoginEpiloguePresenter.kt#L99) to `loginEpilogueView?.siteVerificationFailed(event.site)` so site verification always fails
* Step through the login, and quickly background the app when the site verification dialog appears
* Boom!
* Then pull this branch and notice it no longer crashes

Update release notes:
